Week3/normalization exercise/realistic exercise.md

```diff
@@ -0,0 +1,15 @@
+When a project starts (with or without data in unnormalized form).
+
+The first step is to better your understanding by talking to your client.
+
+"
+_About your data; can you tell me how I should interpret it?
+In other words; how does your department/company work? What do your tasks look like?_

```

Comment:
Are students supposed to answer these question ?
OR ask someone ? or know that these questions are handy ?
I am confused.